### PR TITLE
add jp to mca to nav

### PR DESCRIPTION
**Pull request by lucasogura-tw**
## Context

Adding Japan MCA to nav bar

### Changes

Added JP to nav bar config

## Checklist
- [x] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# v6.0.1
```
Comment (KarlEST) [inline CHANGELOG.md:1]: Should this be minor update instead of patch as we are not fixing a bug?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+## config change: JP is added as SUPPORTED_CARD_LOCALES
```
Comment (KarlEST) [inline CHANGELOG.md:2]: Should it be `SUPPORTED_BORDERLESS_NO_CARD_LOCALES`?

Comment (lucasogura-tw) [reply CHANGELOG.md:2]: good catch

File: CHANGELOG.md (hunk 1 above)
Comment (lucasogura-tw) [reply CHANGELOG.md]: I think you are right, I had just followed this https://github.com/transferwise/public-navigation/commit/10279e1cbded54c9fb47fa4960ad447907d56710

**Review by KarlEST — APPROVED** [review]
I'm not sure how the rules are exactly set up here but if we add it to one place should the `jp` be removed from some other configuration?

**Pull request merged**